core/src/main/scala/org/broadinstitute/dsde/rawls/googleProject/GoogleProjectService.scala

```diff
@@ -85,6 +87,10 @@ class GoogleProjectService(protected val ctx: RawlsRequestContext,
           errorReport = ErrorReport(StatusCodes.Forbidden, "You do not have permission to perform this action.")
         )
       updatedGoogleProject = googleProject.copy(billingAccount = billingProject.billingAccount)
+      _ <- googleServicesDAO.setBillingAccountName(googleProject.googleProjectId,
+                                                   billingProject.billingAccount.get,
+                                                   ctx.toTracingContext
+      )
```

Comment:
What error handling will happen if rawls doesn't have permission on the google billing account to assign billing to a project?  We have the potential case that the google billing account only has the TDR SA added, not the terra billing group (although, we are working to minimize this case as much as possible).

Reply:
I'm not sure, and I'm having a hard time creating that situation to find out!

Reply:
Ok, I've created a free google billing account that hasn't added the terra billing group yet. If you have this deployed on a BEE, I'm glad to test this out. Otherwise, we can leave this as a later task - it's just something we'll want to have in place because I'm having a hard time tracking down all of the billing account owners that have billing profiles only in TDR (and therefore have not granted access to the terra billing group)

Reply:
Sure, I can throw it in a bee - do you need me to add any user to anything or is that something you can handle on your own?

Reply:
I can set up my user!

Reply:
Alright, https://terra.beem2025.bee.envs-terra.bio/ should be ready for you to use then